Source/Injector/Hooks/DarkSouls2/DS2_ReplaceServerAddressHook.cpp

```diff
@@ -46,9 +48,13 @@ bool DS2_ReplaceServerAddressHook::PatchHostname(Injector& injector)
             {
                 continue;
             }
-            if (((info.Protect & PAGE_READWRITE) == 0 && (info.Protect & PAGE_EXECUTE_READWRITE) == 0))
-            {
-                continue;
+            //if i'm not running in wine do the check
+            if (winePrefix == nullptr) {
```

Comment:
couple of coding style issues:
- braces go on next line
- comments start with a space and a capital letter
- variables are in PascalCase not camelCase